{{cookiecutter.repo_name}}/image/config/webapp.conf

```diff
@@ -1,7 +1,19 @@
-server{
+perl_set $port 'sub { return $ENV{"PORT"}; }';
+perl_set $add_cors_headers '
+    sub {
+        my $r = shift;
+        my $origin = $r->header_in("Origin");
+        foreach my $domain (split /[\,\s\|]+/, $ENV{"ALLOWED_HOSTS"}) {
+            return 1 if $origin =~ m|^https?://$domain$|;
+            return 1 if $origin =~ m|^https?://(.*?)\.$domain$|;
+        }
+        return "";
+    }
+';
 
+server {
     #this will be sed changed to the current port by the exec script
-    listen $:PORT:;
+    include /tmp/port;
```

Comment:
Where is this coming from?

Reply:
https://github.com/Lightmatter/lightmatter-base-image/pull/1/commits/f71d9582e09b1c12889d5c81b59ef92c2f5bc3d3